ibc-query/src/core/channel/query.rs

```diff
@@ -281,7 +281,7 @@ where
 
     // Receipt only has one enum
```

Comment:
This line should be removed as we added an additional variant to the `Receipt` enum.